app.py

```diff
@@ -36,6 +45,7 @@
 # set max length to 10.5 MB (one MB larger than peak)
 MAX_LENGTH = int(os.getenv('MAX_LENGTH', 11010048))
 LISTEN_PORT = int(os.getenv('LISTEN_PORT', 8888))
+RETRY_INTERVAL = 5  # seconds
```

Comment:
You want to make this an env variable as well? Changing this would require updating the code as written. I can't really see when we'd want to modify it, so it might not matter that much.

Reply:
We could at least check an env var as an override